mininet/node.py

```diff
@@ -891,6 +894,98 @@ def stop( self ):
         self.deleteIntfs()
 
 
+class LincSwitch(Switch):
+    "User-space LINC-Switch implemented in Erlang."
+
+    configGenCmd = "linc_config_gen {swName} {configArgs}"
+    startCmd = "linc {swName} start"
+    stopCmd = "linc {swName} stop"
+    relCreateCmd = "linc_rel -c {swName}"
+    relDeleteCmd = "linc_rel -d {swName}"
+    listenAddress = "127.0.0.1"
+
+    def _init_(self, name, **kwargs):
```

Comment:
Looks like incorrect method constructor name, should be:
```python
def __init__(self, name, **kwargs):
    super(LincSwitch, self).__init__(name, **kwargs)
```
Or even omit the definition of the constructor if it is calling only the parent one.